Student.js

```diff
@@ -163,5 +163,18 @@ function Student(
         return this.fullName();
     };
 
+    this.checkPreferenceNumbers = function() {
```

Comment:
This method could use a doctag above its signature. See documentation of some other methods for an example.